ci/tasks/upgrade-stemcell.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e
+
+dir=`dirname "$0"`
+source ${dir}/utils.sh
+
+base=$( cd "$( dirname "$( dirname "$0" )")"/.. && pwd )
+base_gopath=$( cd $base/../../../.. && pwd )
+go version
+export GOPATH=$base_gopath:$GOPATH
+echo "GOPATH=" $GOPATH
+
+print_title "INSTALL BOSH CLI..."
+echo "installing bosh CLI"
+gem install bosh_cli --no-ri --no-rdo c
```

Comment:
Started to try v2 on my own, but didn't update it here as the director I'm using for upgrade have not upgrade to use ssl yet.  And moved bosh cli installation to utils.sh, so we can change it later. Thanks.